### DISCUSSION TITLE
To do list

### DISCUSSION THREAD

**Discussion by momysnow** [momysnow/Momy-Desk-Robot#11] [General]
# Code
## Firmware
- [ ] release a stable version
- [x] add code for servo control via driver
- [x] add audio streaming
- [x] simplify all program, creating libraries
## App
- [ ] finish the app server side
- [ ] add gesture control
- [x] add wit.ai api
- [x] add audio streaming

# File 3d
- [x] update model 3d
- [x] add the new electrical components to the project

# Electrical
- [x] update the pin connections in canva

# wiki
- [ ] add setup file
- [ ] add pins
- [x] add libraries
- [ ] add errors more likely
- [ ] add how to use wit.ai
- [ ] add more detailed tutorial of the whole setup

**Comment by benzwd** [comment]
- **Eye States for Functions:**
  - [ ] Focus on visually representing different eye states corresponding to activated functions.

- **Touch Head Animations:**
  - [ ] Explore adding animations triggered by touching the head, potentially displaying weather, time, or stats.

- **Eye Color Customization:**
  - [ ] Integrate a feature to change eye color, considering the background (e.g., white for white PLA), with an option through a dedicated app.
